src/Engine/Combat/CombatDamageModifier.java

```diff
@@ -0,0 +1,22 @@
+package Engine.Combat;
+
+import CommandingOfficers.Commander;
+
+public class CombatDamageModifier extends CombatModifier
+{
+  public int boost;
+  
+  public CombatDamageModifier(Commander user, int ATKboost)
+  {
+    super(user);
+    boost = ATKboost;
+  }
+
```

Comment:
I like to put the @Override annotation just so it's clear this function is from the parent class.